cookbook_se/recipes/04_remote_flyte/lp_notifications.py

```diff
@@ -0,0 +1,109 @@
+"""
+
+06: Getting notifications on workflow termination
+-------------------------------------------------
+
+For background on launch plans, refer to :any:`launch_plans`.
+
+Currently notifications are supported on workflow termination and can be of three flavors:
```

Comment:
Do we just want to point to some other documentation, so that there is just on source of truth?

Reply:
done